exercism/hamming/npm-debug.log

```diff
@@ -0,0 +1,19 @@
+0 info it worked if it ends with ok
```

Comment:
you probably did not intend to check this file in. My guess is you used some variant of `git add .` to create this commit. If so let this be your lesson learned. Always check the contents of your git commits before creating them. Consider getting into the habit of using `git add --patch`